Добавила 4 компонента, 3(Header, Layout, Footer) для дз и 1(HeaderBlo…

sofka opened the pull request:
…ck) во время просмотра видео для дз. Создала для каждого блока module.css

Были затруднения по background, думала нужно img добавить. Потом решила через стили запилить. Как правильно?

sofka committed:
Добавила 4 компонента, 3(Header, Layout, Footer) для дз и 1(HeaderBlock) во время просмотра видео для дз. Создала для каждого блока module.css
Были затруднения по background, думала нужно img добавить. Потом решила через стили запилить. Как правильно?

ZarMarathon commented on src/components/Layout.js:
```diff
@@ -0,0 +1,28 @@
+import layoutStyle from './layout.module.css';
+
+const Layout = ({ id, title, descr, urlBg, colorBg }) => {
+    const backgroundStyle = urlBg ?
+        {
+            background: `url(${urlBg})`,
+
+        } : {
+            backgroundColor: colorBg
+        };
```
В этом условие есть проблема, а что будет если я добавлю и фон и цвет) применятся только фон, а цвет нет))) как бы переписать так что б и фон и цвет добавлялись?)

ZarMarathon reviewed (approved):
В целом ты сделал все правильно, за исключением того что ты положила js файлы сразу в папку components а должна была каждому копоненту создать новую папку))
Исправишь?
А так апрув)) Молодец))) 
Как тебе марафон?

sofka committed:
Переместила файлы в папки соответствующие элементам, файлы переименовала на index.js.
Изменила стили, теперь можно и цвет задать и картинку одновременно.
По марафону: подчеркнула для себя новые фичи, задумалась о покупке оставшейся чати марафона. Подача хорошая, проверка быстрая - это понравилось.

ZarMarathon reviewed (approved):
Вижу что справилась теперь с заданием) Молодец))
